src/util/hass-translation.html

```diff
@@ -54,7 +62,11 @@
 
 window.getTranslation = function (translationInput) {
   const translation = translationInput || getActiveTranslation();
-  const translationFingerprint = window.translationMetadata[translation].fingerprint;
+  const metadata = window.translationMetadata[translation];
+  if (!metadata) {
+    return Promise.reject(new Error('Language ' + translation + ' not found in metadata'));
```

Comment:
Can we implement a similar fallback to `en` here as done below? Showing an error message is a plus, but we don't ever want to let the UI fail to having no strings at all.

https://github.com/home-assistant/home-assistant-polymer/pull/532/files#diff-728281481118ad2715d5d4c85eb8059dR85

Reply:
This can only happen due to dev error, not due to user error.
We end up here if getTranslation is called with unsupported language. Currently the only explicit language is 'en'.

Also, how can we recover from this error? We don't have a fingerprint, so either we don't cache at all killing performance or cache with something like a language making current translation live forever.

Reply:
I don't think it really matters if it's a dev error or user error. That's why the 'en' translation is always included as a fallback. Showing an completely blank UI is the worst thing we could do.

Reply:
Done